keras/initializers.py

```diff
@@ -371,6 +371,28 @@ def he_normal(seed=None):
                            seed=seed)
 
 
+def selu_normal(seed=None):
```

Comment:
I think the question is more: it sounds like these authors have proposed the initialization scheme and therefore it should be called `klambauer_normal` or `hochreiter_normal`?